using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -230,9 +230,37 @@ ACPTarget.getTntId({tntId in
 
 Sends a batch request to your configured Target server for multiple mbox locations that are specified.
 
+{% hint style="warning" %}
+`loadRequests` API is deprecated, and replaced with `retrieveLocationContent` for batch scenarios instead.
+
+Note: When working with prefetch API's, and switching to new `retrieveLocationContent` API, please also use `locationsDisplayed`, otherwise reporting will not work.
+{% endhint %}
+
 {% tabs %}
 {% tab title="Android" %}
-### TargetRequest Builder
+### TargetRequest Constructor
+
+`TargetRequest` Constructor helps create a `TargetRequest` instance. It replaces the deprecated builder pattern. The returned instance can be used with `loadRequests`, which accepts a `TargetRequest` object list to retrieve offers for the specified mbox locations.
+
+#### Syntax
+
+```java
+TargetParameters parameters = new TargetParameters.Builder()
+                                    .parameters(new HashMap<String, String>())
+                                    .profileParameters(new HashMap<String, String>())
+                                    .order(new TargetOrder())
+                                    .product(new TargetProduct())
```

Comment:
Incorrect code, default constructors are not available in TargetOrder and TargetProduct classes.

Reply:
yes, added below as reference
`new TargetOrder(String, Double, List<String>)`
and `new TargetProduct(String, String)`. Please suggest if this is good.